e2e-tests/content-strings/pages/insurance/declarations/anti-bribery/index.js

```diff
@@ -10,66 +10,102 @@ const EXPANDABLE = {
-          'or (b) other than under duress, has been admitted by that person (which shall include, for the avoidance of doubt, admissions of activity made under a deferred prosecution agreement in England, civil forfeiture order or similar administrative settlement in another jurisdiction);',
-          'or (c) is found by a court or competent authority in any competent jurisdiction (after all available rights of appeal have been exhausted) outside the United Kingdom to constitute an offence under any applicable law (except by virtue of changes to that law having retrospective effect);',
-          'or (d) is found by a court or competent authority in the United Kingdom (after all available rights of appeal have been exhausted) to constitute a Relevant Offence and which activity in respect of paragraphs (a), (b) and (c) above corresponds to a Relevant Offence provided that, where for the purposes of paragraph (b) above, any such activity is admitted by an Excluded Person, such activity shall not, for the purposes of this policy, constitute “Corrupt Activity” if such activity was not, at the time it was engaged in, unlawful under the laws or regulations of the country in which it took place;',
+          {
+            TEXT: 'Means any activity with reference to a person (including without limitation the offering of any payment, reward or other advantage to any public official or other person), which is (a) found by a court in a competent jurisdiction (after all available rights of appeal have been exhausted) to have rendered the export contract illegal, void, voidable or unenforceable under its governing law;',
+          },
+          {
+            TEXT: 'or (b) other than under duress, has been admitted by that person (which shall include, for the avoidance of doubt, admissions of activity made under a deferred prosecution agreement in England, civil forfeiture order or similar administrative settlement in another jurisdiction);',
+          },
+          {
+            TEXT: 'or (c) is found by a court or competent authority in any competent jurisdiction (after all available rights of appeal have been exhausted) outside the United Kingdom to constitute an offence under any applicable law (except by virtue of changes to that law having retrospective effect);',
+          },
+          {
+            TEXT: 'or (d) is found by a court or competent authority in the United Kingdom (after all available rights of appeal have been exhausted) to constitute a Relevant Offence and which activity in respect of paragraphs (a), (b) and (c) above corresponds to a Relevant Offence provided that, where for the purposes of paragraph (b) above, any such activity is admitted by an Excluded Person, such activity shall not, for the purposes of this policy, constitute “Corrupt Activity” if such activity was not, at the time it was engaged in, unlawful under the laws or regulations of the country in which it took place;',
```

Comment:
should Corrupt activity use `"` instead of `“`?

Reply:
Good spot, updated.